data/pigui/modules/station-view/07-police.lua

```diff
@@ -1,17 +1,157 @@
+widgetSizes.windowPadding = Vector2(math.ceil(widgetSizes.windowPadding.x),
+									math.ceil(widgetSizes.windowPadding.y))
+
+
+local function payfine(fine)
+	if Game.player:GetMoney() < fine then
+		popup:open()
+		return
+	end
+
+	Game.player:AddMoney(-fine)
+	Game.player:ClearCrimeFine()
+end
+
+
+local function crime_record()
+	local past_crimes, stump = Game.player:GetCrimeRecord()
+	if #utils.build_array(pairs(past_crimes)) > 0 then
+		ui.withFont(orbiteer.xlarge.name, orbiteer.xlarge.size,
+					function() ui.textColored(gray, l.CRIMINAL_RECORD) end)
+		ui.withFont(pionillium.medlarge.name, pionillium.medlarge.size,
+					function ()
+						for k,v in pairs(past_crimes) do
+							ui.textColored(gray, v.count.."\t"..Legal.CrimeType[k].name)
+						end
+					end)
+	end
+end
+
+
+local function outstanding_fines()
+	local crimes, fine = Game.player:GetCrimeOutstanding()
+	if #utils.build_array(pairs(crimes)) > 0 then
+
+		-- headline
+		ui.withFont(orbiteer.xlarge.name, orbiteer.xlarge.size,
+					function() ui.text(l.OUTSTANDING_FINES) end)
+		-- wrap list in medlarge font
+		ui.withFont(pionillium.medlarge.name, pionillium.medlarge.size,
+					function()
+						for k,v in pairs(crimes) do
+							ui.text(v.count.."\t"..Legal.CrimeType[k].name)
+						end
+						local pay_fine_text = string.interp(l.PAY_FINE_OF_N, { amount = Format.Money(fine) })
+						ui.text(pay_fine_text)
+						ui.sameLine()
+						if ui.button(l.PAY, Vector2(100,0)) then
+							payfine(fine)
+						end
+					end)
+	else
+		ui.withFont(pionillium.large.name, pionillium.large.size,
+					function() ui.text(l.WE_HAVE_NO_BUSINESS_WITH_YOU) end)
+	end
+end
+
+
+local function drawPolice()
+
+	-- Todo: add to language translation system
+	local intro_txt = "This is the ".. Game.system.faction.policeName ..
+		" of the "  .. Game.system.faction.name .. "."
+
+	ui.withFont(pionillium.large.name, pionillium.large.size,
+				function () ui.text(intro_txt) end)
+
+	ui.withStyleVars({WindowPadding = widgetSizes.windowPadding,
+					  ItemSpacing = widgetSizes.itemSpacing},
+		function()
+			local infoColumnWidth = ui.getContentRegion().x
+				- widgetSizes.faceSize.x
+				- widgetSizes.windowPadding.x*3
+
+			ui.child("CrimeStats", Vector2(infoColumnWidth, 0),
+					 {"AlwaysUseWindowPadding"},
+					 function()
+						 -- 1. IF OUTSTANDING FINES, SHOW LIST & OFFER TO PAY
+						 outstanding_fines()
+
+						 ui.dummy(Vector2(0,50))
+
+						 -- 2 IF OLD PAYED FINES, SHOW GRAYD OUT LIST
+						 crime_record() end)
+			ui.sameLine()
+			ui.child("PoliceOfficer", Vector2(0, 0),
+					 {"AlwaysUseWindowPadding", "NoScrollbar"},
+					 function ()
+						 if(face ~= nil) then
+							 face:Draw(widgetSizes.faceSize)
+						 end end)
+	end)
+end
+
+
 StationView:registerView({
 	id = "police",
 	name = l.POLICE,
 	icon = ui.theme.icons.shield_other,
-	showView = false,
-	draw = function() end,
-	refresh = function() end,
+	showView = true,
+	draw = drawPolice,
```

Comment:
I usually put the station screens in a wrapper, then you can use the `StationView:shipSummary()` to render the cash/capacity/cabins/legal satus infos at the bottom.

```suggestion
	draw = 
		ui.child("StationPolice", Vector2(0, ui.getContentRegion().y - StationView.style.height), {}, drawPolice)

		StationView:shipSummary()
	end,
```

Reply:
Fixed it, thanks!